.eslintrc.json

```diff
@@ -13,7 +13,9 @@
     "should": true,
     "sinon": true,
     "__VERSION__": true,
-    "__NAME__": true
+    "__NAME__": true,
+    "process": true,
```

Comment:
what is that added for?

Reply:
`karma.config` and `webpack.config` was removed from `.eslintignore` (eslint should lint those files too...) and they are using Node globals that you need to define to eslint so it won't detect them as errors.